debezium-connector-mongodb/src/main/java/io/debezium/connector/mongodb/MongoDbConnectorConfig.java

```diff
@@ -124,9 +125,10 @@ public static SnapshotMode parse(String value, String defaultValue) {
     public static final Field LOGICAL_NAME = Field.create("mongodb.name")
             .withDisplayName("Namespace")
             .withType(Type.STRING)
+            .withGroup(Field.createGroupEntry(Field.Group.CONNECTION, 0))
```

Comment:
The position in group - is it necessary to have it manually assigned? If yes

Are gaps in the allowed?
Are position duplicates allowed?

If an answer to any of the above questions is no then I belive there should be a validation ensuring the rules are not violated.

Reply:
> The position in group - is it necessary to have it manually assigned?

@jpechane, see the discussion above: yes, the manual ordering is required for the time being, as we felt it'd be too disruptive at this point to re-group all the fields. It's something we should do eventually though.

> Are gaps in the allowed?

Yes, that shouldn't matter.

> Are position duplicates allowed?

Ideally not. This sequencing is used for ordering the fields in the Debezium UI, so having duplicates would create an ambiguous situation. So +1 for validating that there are no duplicates, good catch.

@rk3rn3r, can you push a commit for ^^^? Thx!

Reply:
Hmmm currently used positions per group are not stored in a list where I can easily lookup collisions. I thought about resolving collisions on read with the API spec generator. Should I really add it to debezium-core code and log all group/positions in a list and look it up? Maybe I can instead write test in core which gets executed for all connectors that checks all fields of a connector for duplicate group-positions? WDYT?